tests/tick.test.mjs

```diff
@@ -415,8 +415,8 @@ describe('Tick', async () => {
     });
 
     // get the epoch timestamp and assert it is in 24 hours
-    const protocolBalanceAtStartOfEpoch = 50_000_000_0000; // 50M ARIO
```

Comment:
this expectations were actually 0.05% of the balance, but the 50M ARIO value was wrong. so the // 50M ARIO comment was correct, but the //  5% of of the protocol balance and the mARIO number value was incorrect